Assets/RedBlueGames/TextTyper/Examples/TextTyperTester.cs

```diff
@@ -39,12 +37,13 @@ public void Start()
             this.printNextButton.onClick.AddListener(this.HandlePrintNextClicked);
             this.printNoSkipButton.onClick.AddListener(this.HandlePrintNoSkipClicked);
 
-            dialogueLines.Enqueue("Hello! My name is... <delay=0.5>NPC</delay>. Got it, bub?");
+            dialogueLines.Enqueue("Hello! My name is... <delay=0.5>NPC</delay>. Got it, <i>bub</i>?");
             dialogueLines.Enqueue("You can <b>use</b> <i>uGUI</i> <size=40>text</size> <size=20>tag</size> and <color=#ff0000ff>color</color> tag <color=#00ff00ff>like this</color>.");
             dialogueLines.Enqueue("bold <b>text</b> test <b>bold</b> text <b>test</b>");
             dialogueLines.Enqueue("You can <size=40>size 40</size> and <size=20>size 20</size>");
             dialogueLines.Enqueue("You can <color=#ff0000ff>color</color> tag <color=#00ff00ff>like this</color>.");
-            ShowScript();
+            dialogueLines.Enqueue("Sample Shake Animations: <shake=lightrot>Light Rotation</shake>, <shake=lightpos>Light Position</shake>, <shake=fullshake>Full Shake</shake>\nSample Curve Animations: <curve=slowsine>Slow Sine</curve>, <curve=bounce>Bounce Bounce</curve>, <curve=crazyflip>Crazy Flip</curve>");
```

Comment:
Thinking the user shouldn't care if it's a shake or a curve - that's an implementation detail that the person writing this text may not know.

So we should try to get the syntax to be the same. Thinking we can make it "anim=XXX" and have it look it up in both libraries to figure out whether it's a shake or a curve.